src/site/content/ru/fast/remove-unused-code/index.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post-old
+title: Удаление неиспользуемого кода
+subhead: |
+  Npm упрощает добавление кода в проект. Но так уж ли нужны все эти лишние байты?
+authors:
+  - houssein
+date: 2018-11-05
+description: |
+  Реестры вроде npm изменили экосистему JavaScript к лучшему: они позволяют легко скачать и использовать более полумиллиона общедоступных пакетов. Но часто мы добавляем библиотеки, которые используем не полностью. Поэтому нужно анализировать пакеты и находить неиспользуемый код.
+codelabs:
+  - codelab-remove-unused-code
+tags:
+  - performance
+---
+
+Реестры вроде [npm](https://docs.npmjs.com/getting-started/what-is-npm) изменили
+экосистему JavaScript к лучшему: они позволяют легко скачать и использовать
+более _полумиллиона_ общедоступных пакетов. Но часто мы добавляем библиотеки,
+которые используем не полностью. Поэтому нужно **анализировать пакеты**
+и находить неиспользуемый код, а затем удалять **неиспользуемые** и **ненужные** библиотеки.
+
+## Анализ пакета
+
+С помощью DevTools можно легко узнать размер сетевых запросов:
+{% Instruction 'devtools-network', 'ol' %}
+{% Instruction 'disable-cache', 'ol' %}
+{% Instruction 'reload-page', 'ol' %}
+
+{% Img src="image/admin/aq6QZj5p4KTuaWnUJnLC.png", alt="Панель «Сеть» (Network) с запросом пакета", width="800", height="169", class="w-screenshot" %}
+
+Вкладка [Покрытие](https://developer.chrome.com/docs/devtools/coverage/)
+(Coverage) в DevTools также сообщает, сколько CSS- и JS-кода в приложении
+не используется.
+
+{% Img src="image/admin/xlPdOMaeykJhYqGcaMJr.png", alt="Покрытие кода в DevTools", width="800", height="562", class="w-screenshot w-screenshot--filled" %}
+
+Указав полную конфигурацию Lighthouse через Node CLI, с помощью аудита
+«Unused JavaScript» (Неиспользуемый JavaScript) можно отслеживать, сколько
+неиспользуемого кода поставляется с приложением.
+
+{% Img src="image/admin/tdC0d65gEIiHZy6eyo82.png", alt="Аудит «Unused JavaScript» в Lighthouse", width="800", height="347", class="w-screenshot" %}
+
+Если в качестве упаковщика вы используете [webpack](https://webpack.js.org/),
+то разобраться в том, что входит в комплект, вам поможет
+[Webpack Bundle Analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer). Включите этот плагин в файл
```

Comment:
`Включите` can be misunderstood as "turn on" instead of "include". I'd use "добавить", which in turn can be misread as "add", but it less confusing for readers.

```suggestion
[Webpack Bundle Analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer). Добавьте этот плагин в файл
```

Reply:
affirmative